app/src/main/java/org/kinecosystem/kinit/network/TaskService.kt

```diff
@@ -64,6 +63,309 @@ class TaskService(context: Context, api: TasksApi,
         })
     }
 
+    val DummyTask = """
+{
+  "id": "0",
+  "start_date": 1532252893,
+  "provider": {
+    "name": "Kinit Team",
+    "image_url": "https://cdn.kinitapp.com/brand_img/poll_logo_kin.png"
+  },
+  "min_to_complete": 2,
+  "memo": "1-kit-sb0ea956eabef4705bd26b",
+  "tags": [
+    "Finance"
+  ],
+  "type": "quiz",
+  "title": "Kin Quiz",
+  "price": 50,
+  "desc": "Let's test your Kinowledge!",
+  "updated_at": 1531315377,
+  "items": [
+    {
+      "id": "844",
+      "results": [
+        {
+          "id": "46584",
+          "text": "I know that already (Correct)"
+        },
+{
+          "id": "46585",
+          "text": "wrong I know that already"
+        },
+{
+          "id": "46586",
+          "text": "I am not sure"
+        },
+        {
+          "id": "4589",
+          "text": "I'm still a bit confused about that"
+        }
+      ],
+      "type": "text",
+      "text": "Kin is a cryptocurrency that can be earned or spent on things you like.",
+      "quiz_data": {
+        "answer_id": "46584",
+        "explanation": "Kin is a cryptocurrency that can be earned or spent on things you like ecosystem of digital services.",
+        "reward": 5
+      }
+    },
+    {
+      "id": "347",
+      "results": [
+        {
+          "id": "4359698",
+          "text": "For investing my time"
+        },
+        {
+"id": "36756",
+          "text": "All of the God (Correct)"
+        },
+        {
+          "id": "45763",
+          "text": "For creating value"
+        },
+        {
+          "id": "345768",
+          "text": "For participating in the Kin Ecosystem"
+        }
+      ],
+      "type": "text",
+      "text": "Why are you getting Kin?",
+      "quiz_data": {
+        "answer_id": "36756",
+        "explanation": "you getting Kin from God",
+        "reward": 5
+      }
+    },
+    {
+      "id": "372",
+      "results": [
+{
+          "id": "342756",
+          "text": "wrobnf Got it!"
```

Comment:
spelling

Reply:
@hello8721 thanks for your review! However, I believe what you're seeing is just temporary, throw-away code. this content will eventually come from the server and will not be hard coded in the client. its just until the feature is supported in the server (read: until I do my work :] )

Reply:
@shayb this shouldn't be merged to master